### ISSUE TITLE
BlockchainSynchronizer

### ISSUE THREAD

**Issue by bodzie** [seredat/karbowanec#8]
2017-Jul-23 15:12:31.807775 ERROR   [BlockchainSynchronizer] Failed to query outdated pool transaction: NodeErrorCategory:3, Network error

I have this error in wallet, help pls

**Comment by seredat** [comment]
First let daemon sync, then launch wallet. Let us know if you suceed.

**Comment by bodzie** [comment]
In the beginning I run the daemon, then I start the wallet, but it does not solve the problem, what else could be the problem?

**Comment by aivve** [comment]
Run the daemon then read what it sais. Run wallet when it sais 'You are now
synchronized with the network. You may now start simplewallet.'

**Comment by bodzie** [comment]
The fact is that I'm now trying to create my own Crypto currency, based on the Carbowanec, if it's not hard for you to tell me what the problem may be, I edited the config, maybe there's a problem in checkpoints? Thank you so much (https://github.com/bodzie/talentum/blob/master/src/CryptoNoteConfig.h)

**Comment by seredat** [comment]
There are detailed guides how to fork CryptoNote currecy, refer for example to Forknote or https://cryptonotestarter.org 
Checkpoints and other Karbowanec related stuff, like default ports should be changed or removed

**Comment by bodzie** [comment]
I understand, tkank you so much

**Comment by bodzie** [comment]
Are there any restrictions on the version of ubuntu?

**Comment by seredat** [comment]
For what? On 14.04 it will be most easy to get it compiling and working. It should work on other versions too.

**Comment by bodzie** [comment]
Well, I'll try to run it on 16.04, thx

**Comment by bodzie** [comment]
How can I make a build for windows? For windows it is possible?

**Comment by markotdel** [comment]
Greetings. Same problem. I changed the VPS server. Accordingly, IP servers also became different. I edited CryptoNoteConfig.h, I registered new IP. Poured into new servers, launched daemon on each. But they do not see each other. Already tried to change the port. I checked the connections, the addresses are pinged. Naturally, the purse sends an error "[BlockchainSynchronizer] Failed to query outdated pool transaction: NodeErrorCategory: 3, Network error". What is the reason I can not understand. I ask for help.

**Comment by aivve** [comment]
Trying to ask for help from Ukrainian devs on making clone called "putcoin"? :) Lovely.
I recommend goto Forknote, you will not need to build anything, just edit config and youre good to go. 
Don't fork Karbo, guys, there are easier ways.

**Comment by markotdel** [comment]
This is a joke, PutinCoin will not be popular. :) If suddenly I offended my Ukrainian colleagues with a joke, I apologize.
As for the case, I used for My Coin https://github.com/cryptonotefoundation/cryptonote. Everything worked well, the mining went, the synchronization. I tried to transfer the data to new servers, this caused the problem. Demons do not see each other. The print_cn command does not have connections.
